config/application.yml.default

```diff
@@ -49,6 +49,13 @@ arcgis_api_generate_token_url: 'https://gis.gsa.gov/portal/sharing/rest/generate
 arcgis_api_suggest_url: 'https://gis.gsa.gov/servernh/rest/services/GSA/USA/GeocodeServer/suggest'
 arcgis_api_find_address_candidates_url: 'https://gis.gsa.gov/servernh/rest/services/GSA/USA/GeocodeServer/findAddressCandidates'
 arcgis_api_request_timeout_seconds: 5
+arcgis_api_token_prefetch_ttl: 10
+arcgis_api_token_cache_key_prefix: arcgis_api_token
```

Comment:
does this need to be a config if we don't intend to change it based on the env?

Reply:
@svalexander added based on Tim's comment

Reply:
@svalexander Keeping the cache key prefixes in config makes them more visible, helps prevent name conflicts, and can help distinguish what is and isn't in cache. Might be good to consolidate these into a specific subsection of config eventually.